qutebrowser/browser/commands.py

```diff
@@ -1960,6 +1961,14 @@ def jseval_cb(out):
                         out = out[:5000] + ' [...trimmed...]'
                     message.info(out)
 
+        if file:
+            try:
+                with open(js_code, 'r', encoding='utf-8') as f:
+                    js_code = f.read()
+            except OSError as e:
+                message.error(str(e))
+                return
```

Comment:
I didn't see this earlier, but you can actually do `raise cmdexc.CommandError(str(e))` here.